app/media_sources/facebook_media_source.rb

```diff
@@ -61,11 +61,34 @@ def initialize(url)
   # @return [Forki::Post]
   def retrieve_facebook_post
     # Unlike Zorki, Forki expects a full URL
-    # begin
-      Forki::Post.lookup(url)
-    # rescue NoMethodError => e
-    #   debugger
-    # end
+    posts = Forki::Post.lookup(url)
+    self.class.create_aws_key_functions_for_posts(posts)
+
+    return posts if Figaro.env.AWS_REGION.blank?
+
+    posts.map do |post|
+      @@logger.debug "Beginning uploading of files to S3 bucket #{Figaro.env.AWS_S3_BUCKET_NAME}"
+
+      # Let's see if it's a video or images, and upload them
+      if post.image_file.blank? == false
```

Comment:
Can't we use `present?` here (and elsewhere that we're doing `.blank? == false`)

Reply:
And today I learned something.